olive/model/utils/hf_utils.py

```diff
@@ -188,3 +189,11 @@ def get_model_max_length(model_name: str, fail_on_not_found=False) -> int:
             else:
                 logger.warning(not_found_msg)
                 return None
+
+
+def get_loading_args(hf_config: HfConfig):
```

Comment:
we need move this function into HfConfigMixin

Reply:
The function will have to be static in a mixin. Mixins are meant to be subclassed and using it merely for assessing static functions in other files seems like bad use case. If the use case was only in the subclasses, I would agree but that's not the case here. This function does belong in utils.

Reply:
I'd like the HfConfig decouple with hf_utils. From the name, the HfConfig is concept of model. On another hand, hf_utils is collected utils function invoking the transformer package.

If we'd like to make these functions as common functions, my suggestion is to create another util above both HfConfig and existing hf_utils. 

Another candidate is we could move the get_loading_args to HfConfig class since it has not any dependencies to transformers. Then we only need define one function 
```
def get_model_type(model_name, loading_args) -> str:
```

How do you think? @jambayk

Reply:
Fair enough. Pulled the relevant functions into `hfconfig_utils.py`.